frontend/styles/index.scss

```diff
@@ -6,3 +6,5 @@
 @import "~tailwindcss/base";
 @import "~tailwindcss/components";
 @import "~tailwindcss/utilities";
+
+[x-cloak] { display: none; }
```

Comment:
gebruik je dit? Heeft tailwind hier niet iets voor?

Reply:
https://ryangjchandler.co.uk/articles/hiding-elements-until-alpine-is-ready-with-x-cloak